kuzzleio/kuzzle-admin-console
# [HOTFIX] Render search results with undefined meta

## What does this PR do ?
Fixes issue that broke document view when some of the documents had undefined `_meta` field.

### How should this be manually tested?
Serve some documents with the `_meta` field set to `undefined` (you can use a pipe plugin in your Kuzzle backend) and see them in the documents page.

### Other changes
Show the error traces after showing the toast.
